docs/docs/20-usage/45-cron.md

```diff
@@ -0,0 +1,37 @@
+# Cron
+
+To configure cron jobs you need at least push access to the repo.
+
+:::warning
+By default pipelines triggered by cron jobs wont execute any steps in pipelines, as they are not part of the default event filter and you explicitly need to set a `event: cron` filter.
```

Comment:
Should we stick to `crons` instead of `cron jobs`?

Reply:
well I dont have a strong opinion on this one

Reply:
I rename all exitencens in code to be `cron` so they are much easier to recognize IMO. For comments, descriptios and docs I left `cron job` sometimes as it was nice to read. Hope you are fine with that change.